### DISCUSSION TITLE
New Website Appearance Setting In Firefox Nightly 100

### DISCUSSION THREAD

**Discussion by ghost** [darkreader/darkreader#8489] [General]
Hey @Gusted,

I just started noticing this new setting in Firefox Nightly 100 today. I just wanted to post a friendly notification about it.

I don't know if this has any effect or bearing on Dark Reader. Maybe this will hinder / help Dark Reader? Just curious.

![image](https://user-images.githubusercontent.com/95879668/159491723-e90601f6-5925-406b-b7ab-46e5e636b211.png)

Maybe Mozilla is encroaching on Dark Reader's territory slowly, but surely?!? Taking over the world... One extension at a time!!! 🤣🤣🤣

**Comment by Gusted** [comment]
Ah they've finally made the UI for it(the main code for this was merged a few months ago and was only accessible via `about:config`), I'm not much intrigued by this implementation, it's similar to how one of the chromium's dark mode operate and in my biased opinion it doesn't offer much of a replacement, it will do it's job on 50% of the sites, but the other 50% of the sites wouldn't be that good looking.

**Reply by Gusted** [reply]
Thanks for the friendly notification btw. Oh, it shouldn't interfere _hopefully_.

**Reply by ghost** [reply]
You're Welcome! 😄

I was thinking along the same lines as you with the Chrome dark mode. If I remember correctly, I thought at one time they had a force dark mode experimental flag or something like that. I believe it was there for a few versions and then it turned out to be complete crap or really buggy and they removed it? My memory is a little fuzzy on all the details of it, but that's what I recall.
